Sources/Datadog/Utils/InternalLoggers.swift

```diff
@@ -27,49 +17,44 @@ internal var userLogger = createNoOpSDKUserLogger()
 
 internal func createNoOpSDKUserLogger() -> Logger {
     return Logger(
-        logBuilder: nil,
-        logOutput: nil,
-        dateProvider: SystemDateProvider(),
+        core: NOOPDatadogCore(),
         identifier: "no-op",
+        serviceName: nil,
+        loggerName: nil,
+        sendNetworkInfo: false,
+        useCoreOutput: false,
+        validation: nil,
         rumContextIntegration: nil,
-        activeSpanIntegration: nil
+        activeSpanIntegration: nil,
+        additionalOutput: nil,
+        logEventMapper: nil
     )
 }
 
 internal func createSDKUserLogger(
-    configuration: InternalLoggerConfiguration,
+    in core: DatadogCoreProtocol,
```

Comment:
The `userLogger` now need a `core` instance with a registered `LoggingFeature` instance :/
An alternative would be to provide 2 different implementations of a Logger, one that uses `core`, and one that write logs directly to the console without `core`. LMKWYT!

Reply:
Makes sense to have separate implementation for core logger. After all, we will need something self-contained that will live solely in Core module, so we can't use `Logger` nor even `LogOutput` in there. I think that a thin `CoreLogger` construct can make sense here, with not much "outputs" abstraction and a simple `print()` at at the bottom 👌.

Reply:
Also, a question is: should the _core logger_ be specific to an instance of the `core` or can we have a single _core logger_ for all SDK instances? I think the prior makes more sense, as messages could be annotated with the SDK instance name. Otherwise, internal logs could be ambiguous.